web/explorer/src/views/ImportView.vue

```diff
@@ -15,9 +15,18 @@ import { watch } from "vue";
 import DescriptionPanel from "@/components/DescriptionPanel.vue";
 import UploadOptions from "@/components/UploadOptions.vue";
 
-// import demo data
-import demoRdocStatic from "@testfiles/rd/al-khaser_x64.exe_.json";
-import demoRdocDynamic from "@testfiles/rd/0000a65749f5902c4d82ffa701198038f0b4870b00a27cfca109f8f933476d82.json";
+const isBundle = import.meta.env.MODE === "bundle";
+let demoRdocStatic = null;
+let demoRdocDynamic = null;
+
+if (!isBundle) {
+    import("@testfiles/rd/al-khaser_x64.exe_.json").then((module) => {
+        demoRdocStatic = module.default;
+    });
+    import("@testfiles/rd/0000a65749f5902c4d82ffa701198038f0b4870b00a27cfca109f8f933476d82.json.gz").then((module) => {
+        demoRdocDynamic = module.default;
+    });
+}
```

Comment:
rather than putting this directly in the bundle, I recommend that the demo simply redirect to a URL with the `rdoc=` parameter pointing to a static file on GH Pages. this way, users don't have to download the result document content unless they want to see the demo.

Reply:
in fact, the buttons could be literal `<a>` elements without any JS handlers.

Reply:
As advised, the buttons now redirect the user to `/?rdoc=<URL>` where `<URL>` points to capa-testfiles Github repo. For dynamic, I relaxed the version back to `6.1.0` because `?rdoc` query only supports loading JSON resources.

Test [here](https://s-ff.github.io/capa/explorer).

Reply:
demo works great! and i'm impressed by how quickly it loads. nice work :-)

Reply:
> because ?rdoc query only supports loading JSON resources.

meaning no compressed `json.gz` files?

Reply:
as discussed, let's add this functionality to load compressed files (we can then store gz files in our testfiles repo) and link to it from capa explorer web.

Reply:
@mr-tz should be completed with latest commits.